Sources/App/Commands/Analyzer.swift

```diff
@@ -213,19 +213,33 @@ func reconcileVersions(application: Application,
     }
 
     let references = defaultBranch.and(tags).map { $0 + $1 }
-    let versions: EventLoopFuture<[Version]> = references
+    let incoming: EventLoopFuture<[Version]> = references
         .flatMapEachThrowing { ref in
             let revInfo = try Git.revisionInfo(ref, at: cacheDir)
             return try Version(package: package,
                                reference: ref,
                                commit: revInfo.commit,
                                commitDate: revInfo.date) }
 
-    let delete = Version.query(on: transaction)
-        .filter(\.$package.$id == pkgId)
-        .delete()
-    let insert = versions.flatMap { versions in versions.create(on: transaction).map { versions }  }
-    return delete.flatMap { insert }
+    let newReconciliation = true
+    if newReconciliation {
```

Comment:
I've feature flagged this for local testing but this may actually be something we might want to consider having, since this is a rather critical part of the analysis stage. We could leave it in for a while and then remove the code in a couple of days. (I wouldn't want to leave this in for too long.)

On the other hand we can also simply revert the PR. This is not something that's hard to roll back even if there was a merge conflict.

I do like how minimal the actual change to the processing is.

Reply:
I think I'm leaning towards removing that flag and letting it run on staging for half a day.

Reply:
Yep, let's take it out and run it on staging for a few hours.